docs/github/v2/04-builder.md

```diff
@@ -313,6 +313,8 @@ Parameters must start with a hyphen (`-`) and may be followed by a value (withou
 
 Parameters without a value will be considered booleans (with a value of true).
 
+Complete list of available parameters can be found in the [Unity Documentation](https://docs.unity3d.com/ScriptReference/BuildOptions.html).
```

Comment:
This is currently confusing editor parameters with parameters from our API, we should be clear about that these can only be passed through custom-parameters parameter from our API

Reply:
This line of documentation is located within the subsection for the `customParameters` parameter.
Does the location not make it clear enough?
Thoughts on how it could be made clearer?

Reply:
Sure, I can think of two examples of how it could be made more explicit:

- We could name the different uses of customParameters, i.e. for passing into your script as well as use them as build options as described in that link.
- Explicitly call `BuildOptions` by it's name: Editor's `BuildOptions`.